src/client/css/components/_search.scss

```diff
@@ -9,5 +9,21 @@
 
     &__grid-tile {
         max-width: 200px;
+
```

Comment:
I think needing to do things like this is precisely why the Material UI folks have moved away from inlining all their styles